src/components/_classes/component/Component.js

```diff
@@ -2590,6 +2590,13 @@ export default class Component extends Element {
       !this.isEmpty(this.defaultValue) &&
       this.isEqual(this.defaultValue, this.dataValue);
 
+    // We want to always validate calculated values...
+    //  .. unless they haven't recently changed, and are NaN, which is frequently the case when the form is empty.
+    if (this.calculatedValue !== undefined && !(Number.isNaN(this.calculatedValue) && !calcChanged)) {
```

Comment:
I am ok with this change as long as we do not make an assumption that calculated values should be numbers. You can create a calculated value as any data type. The name should really be computedValue